LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Borys Minaiev
```

Comment:
I don't know what type of license we want / which name we want to put here, but probably not mine (at least because Egor wrote most of the code here). Maybe @EgorKulikov has opinions about what to put here?